extensions/ingestion/utils/MongoMDWrapper.js

```diff
@@ -0,0 +1,43 @@
+const arsenal = require('arsenal');
+const Logger = require('werelogs').Logger;
+
+const { MongoClientInterface } = arsenal.storage.metadata.mongoclient;
+
+const USERSBUCKET = arsenal.constants.usersBucket;
+const PENSIEVE = 'PENSIEVE';
+const METASTORE = '__metastore';
+
+class MongoMDWrapper extends MongoClientInterface {
```

Comment:
I really only need the constructor and setup methods. If I were to try and create a new class without extending though, I'd have to add the mongodb module to this repo for this one use case